eo-compiler/src/main/antlr4/org/eolang/compiler/Program.g4

```diff
@@ -78,12 +155,11 @@ arguments_declaration returns [List<Argument> ret]
 
 argument_declaration returns [Argument ret]
     :
-    HINAME
-    SPACE
-    LONAME
+    HINAME SPACE LONAME
```

Comment:
@g4s8 why did you change the formatting? I think we should keep tokens one per line, like everywhere else in this file.